plugins/jebscape

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/JebScape/JebScape-Client.git
+commit=446ec696fb5e839010e4c91cac5693215dc4982c
```

Comment:
Standard disclaimer used for plugins that send data to third-party server

```suggestion
commit=446ec696fb5e839010e4c91cac5693215dc4982c
warning=This plugin sends player data, including usernames, and IP address to a third-party server for communication between clients.
```

Reply:
Thanks for the suggestion. I did add a similar disclaimer to the public description but replaced username since their login username isn’t sent. Account hashes and display names are. Could we reword the suggestion slightly to match my description?
“This plugin submits your unique account hash and IP address to a 3rd party server not controlled or verified by the RuneLite Developers.”

Maybe we could add display name to that list.

Reply:
Feel free (I'm not affiliated with RuneLite so I don't make approve/deny decisions)

Reply:
I do like your final few words “for communication between clients.”

Reply:
Oh okay.